README.md

```diff
@@ -29,26 +17,22 @@ Join discord server : https://discord.gg/t6rC5RaJQn
 * Optimized for M1/M2 Chips
 * Runs locally on your computer
 
+## Requirements 
+* M1/M2 Mac
+* 16 GB RAM preferred. It will run a bit slow with 8 GB ram.
+* MacOS 12.5.1 or later
 
-## How to use 
-
-1) Download and start the application
-
-2) Enter the text prompt.
-<img width="1162" alt="Screen Shot 2022-09-11 at 12 33 52 PM" src="https://user-images.githubusercontent.com/1890549/189538839-45ac91b1-cd66-4543-9ece-956220d0d769.png">
-
-3) Wait a few seconds to get the generated image
-
-![image](https://user-images.githubusercontent.com/1890549/189539684-222482fb-63f7-4799-bfc1-005b84508e35.png)
+## License
+Stable Diffusion is released under [the CreativeML OpenRAIL M license](https://github.com/CompVis/stable-diffusion/blob/main/LICENSE).
+Diffusion Bee is just a GUI wrapper on top of Stable Diffusion, so all the term of Stable Diffusion are applied on the outputs.
 
+## How to use
```

Comment:
Change to "Usage", this is a lot clearer and inline with a lot of other documentation I have seen and worked on.